frontend/scripts/generateDocs.ts

```diff
@@ -1,24 +1,41 @@
 import fs from 'fs'
 import path from 'path'
+import { execSync } from 'child_process'
+
+const PACAKGE_NAME = '@acaldas/document-model-libs'
+
+try {
+  execSync(`npm ls ${PACAKGE_NAME}`)
+} catch (e) {
+  throw new Error(`Failed to generate documentation from the dependency (${PACAKGE_NAME}), did you run \`npm install\`?`)
+}
+
+// eslint-disable-next-line import/first
 import { concatMdSync } from 'concat-md'
 
-const DOCS_DIR = process.env.DOCS_DIR
+const DOCS_DIR = path.resolve(__dirname, '..', 'node_modules', '@acaldas', 'document-model-libs', 'dist', 'docs')
+const DOCS_VERSION = execSync(`npm view ${PACAKGE_NAME} version`).toString()
 
 function generateMdDocs (pathToDir: string, outputFilePath: string) {
   const content = concatMdSync(pathToDir, {
     fileNameAsTitle: true
   })
-  fs.mkdirSync(path.dirname(outputFilePath))
+  const dirname = path.dirname(outputFilePath)
+  if (!fs.existsSync(dirname)) {
+    fs.mkdirSync(dirname, { recursive: true })
+  }
   fs.writeFileSync(outputFilePath, content, { encoding: 'utf8' })
 }
 
 function main () {
-  if (!DOCS_DIR) {
-    throw new Error('Please provide DOCS_DIR env variable which points to documentation directory')
+  const dirname = path.resolve(__dirname, '..', 'content', 'documentation')
+  const outputFilePath = path.resolve(dirname, 'index.md')
+  const outputVersionPath = path.resolve(dirname, '.version')
+  if (!fs.existsSync(outputVersionPath) || fs.readFileSync(outputVersionPath, { encoding: 'utf8' }).toString() !== DOCS_VERSION) {
```

Comment:
We only want to regenerate documentation if it does not exist or if the package version is not the same as the one the documentation is generated for. 

It does not seem reasonable to always regenerate the docs.

Reply:
Then, I would also move writing of `DOCS_VERSION` into the `generateMdDocs` function. Otherwise, now, the version file can be updated without updating the docs